tests/e2e/monitoring/test_monitoring_pvc_expansion.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import tier4
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.resources.pvc import get_pvc_obj
+from ocs_ci.ocs.monitoring import prometheus_health_check
+
+logger = logging.getLogger(__name__)
+
+
+@tier4
```

Comment:
I don't think it should be considered as tier4. The test case is not disruptive. I would put it in `tier2` or `workloads`

Reply:
+1
This can be in workloads